code/modules/mob/living/simple_animal/hostile/megafauna/onestarboss.dm

```diff
@@ -541,23 +460,23 @@ Effects demand parent code, so ..()
 									step(B, SOUTH)
 									step(A, NORTH)
 									step(B, SOUTH)
-							spawn(80 MILLISECONDS)
+							spawn(12)
 								step_towards(A, target_location)
 								step_towards(B, target_location)
-							spawn(120 MILLISECONDS)
+							spawn(16)
 								step_towards(A, target_location)
 								step_towards(B, target_location)
-							spawn(160 MILLISECONDS)
+							spawn(20)
 								step_towards(A, target_location)
 								step_towards(B, target_location)
 							var/i = 15
 							playsound(loc, 'sound/machines/onestar/boss/minigun_windup.ogg', 75)
-							sleep(500 MILLISECONDS)
+							sleep(10)
 							while(i != 0 && src.stat != DEAD)
 								i--
 								shoot_minigun(A)
 								shoot_minigun(B)
-								sleep(500 MILLISECONDS)
+								sleep(1)
```

Comment:
Is there any reason why you changed the argument for sleep to be direct numbers instead of using the time macros?